engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/views/register.html

```diff
@@ -15,17 +15,21 @@ <h2 translate>Content Host Registration</h2>
       </li>
       <li>
         <p translate>Install the pre-built bootstrap RPM:</p>
-        <pre><code>rpm -Uvh http://{{ noCapsulesFound ? katelloHostname : hostname(selectedCapsule.url) }}/pub/{{ consumerCertRPM }}</code></pre>
+        <pre><code>rpm -Uvh https://{{ noCapsulesFound ? katelloHostname : hostname(selectedCapsule.url) }}/pub/{{ consumerCertRPM }}</code></pre>
```

Comment:
:+1:

Reply:
Not required here, but I think this benefits from changing to `yum install` as that ensures all dependencies are installed alongside the consumer RPM.